app/tests/e2e/test_tenants.py

```diff
@@ -251,13 +252,13 @@ async def test_update_tenant_ecosystem_verifier_to_issuer(
     assert_that(tenant).has_tenant_name(name)
     assert_that(tenant).has_created_at(wallet.created_at)
     assert_that(tenant).has_updated_at(wallet.updated_at)
-    assert wallet.settings["wallet.name"].startswith("ecosystem.")
+    assert_that(wallet.settings["wallet.name"]).is_type_of(str)
```

Comment:
same as above - possibly specific check for length and/or format instead of type

Reply:
This has been addressed in #232